src/test/java/teammates/test/pageobjects/InstructorCommentsPage.java

```diff
@@ -208,6 +208,69 @@ public void clickResponseCommentDelete(int sessionIdx, int questionIdx, int resp
         waitForPageToLoad();
     }
     
+    /**
+     * Clicks all the headings of the comment panel to either expand / collapse the panel body.
+     */
+    public void clickAllCommentsPanelHeading() {
+        for (WebElement e : browser.driver.findElements(By.cssSelector("div[id^='panel_display-']"))) {
+            e.findElement(By.cssSelector(".panel-heading")).click();
+        }
+    }
+    
+    /**
+     * Check if the body of all the comment panel is indeed expanded.
+     * @return true if all comment panel body is expanded, otherwise false.
+     */
+    public boolean isAllCommentsPanelBodyExpanded() {
+        By panelCollapseSelector = By.cssSelector(".panel-heading+.panel-collapse");
+        List<WebElement> webElements = browser.driver.findElements(panelCollapseSelector);
+        
+        for (WebElement e : webElements) {
+            if (!e.isDisplayed()) {
+                return false;
+            }
+        }
+        
+        return true;
+    }
+
+    
+    /**
+     * Check if the body of all the comment panel is indeed collapsed.
+     * @return true if all comment panel body is collapsed, otherwise false.
+     */
+    public boolean isAllCommentsPanelBodyCollapsed() {
+        By panelCollapseSelector = By.cssSelector(".panel-heading+.panel-collapse");
+        List<WebElement> webElements = browser.driver.findElements(panelCollapseSelector);
+        
+        for (WebElement e : webElements) {
+            if (e.isDisplayed()) {
+                return false;
+            }
+        }
+        
+        return true;
+    }
+    
+    /**
+     * Method to wait for all the panels to collapse. Typically called after clicking panel headings.
```

Comment:
To follow the commenting style of other methods, `Waits for all panels to ...`
~~`Typically called after clicking panel headings.`~~ (Avoid referring to where the method is used, but you can give an example of how it can be used. Reason: This method has no control over where it can be used.).